apps/src/StudioApp.js

```diff
@@ -1269,7 +1294,7 @@ StudioApp.prototype.onMouseMoveVizResizeBar = function (event) {
  * Resize the visualization to the given width
  */
 StudioApp.prototype.resizeVisualization = function (width) {
-  var codeWorkspace = document.getElementById('codeWorkspace');
+  var workspaceRight = $(".workspace-right");
```

Comment:
I don't think naming things according to their LTR layout is necessarily harmful.  That said, I've been thinking of them as 'visualization column' and 'editor column'.  They contain other things, but that description seems to hold up in most cases.